Sources/Sentry/SentryScope.m

```diff
@@ -460,13 +460,16 @@ - (SentryEvent *__nullable)applyToEvent:(SentryEvent *)event
     @synchronized(_spanLock) {
         span = self.span;
     }
+    if (![event.type isEqualToString:SentryEnvelopeItemTypeTransaction])
+        if ([span isKindOfClass:[SentryTracer class]])
+            event.transaction = [(SentryTracer *)span name];
+    if (newContext == nil)
+        newContext = [NSMutableDictionary new];
+    newContext[@"trace"] = [span serialize];
+}
 
-    if (![event.type isEqualToString:SentryEnvelopeItemTypeTransaction] &&
-        [span isKindOfClass:[SentryTracer class]]) {
-        event.transaction = [(SentryTracer *)span name];
-    }
-
-    return event;
+event.context = newContext;
+return event;
```

Comment:
`h`: I think there is something broken here.

Reply:
Why do you think that?

Reply:
Big WOW. I wondering how this didn't show on compilation.